docs/sdks/uid2-sdk-ref-java.md

```diff
@@ -12,16 +12,22 @@ You can use UID2 server-side SDKs to facilitate decrypting of UID2 advertising t
 <!-- This guide includes the following information:
 
 - [Overview](#overview)
+- [Audience](#audience)
```

Comment:
There is no similar change for other SDKs, is this intentional?

Reply:
Yes -- for the moment. I have that info for the Java SDK, from the readme. The other readmes are very sparse. Hoping to add similar info but I don't currently have it. Happy to include whatever is available.